### PR TITLE
fix: Unable to get changelogs when using GitHub API

### PR DESCRIPTION
**Pull request by TheAabedKhan**

### CODE REVIEW

File: lib/services/manager_api.dart [hunk 1]
```diff
@@ -115,6 +119,9 @@ class ManagerAPI {
   }
 
   String getPatchesRepo() {
+    if (!isUsingAlternativeSources()) {
+      return defaultPatchesRepo;
+    }
```
Comment (kitadai31) [inline lib/services/manager_api.dart:124]: Shouldn't `getIntegrationsRepo()` also be changed?

File: lib/services/manager_api.dart [hunk 2]
```diff
@@ -115,6 +119,9 @@ class ManagerAPI {
...
     return _prefs.getString('patchesRepo') ?? defaultPatchesRepo;
```
Comment (Ushie) [inline lib/services/manager_api.dart:125]: Is that `??` necessary?

Comment (ILoveOpenSourceApplications) [reply lib/services/manager_api.dart:125]: > Is that `??` necessary?

@TheAabedKhan.

Comment (TheAabedKhan) [reply lib/services/manager_api.dart:125]: Not needed after committing that logic.

**Comment by ILoveOpenSourceApplications** [comment]
Is this PR ready?

**Comment by TheAabedKhan** [comment]
@ILoveOpenSourceApplications YES! Ready to be merged.

**Review by oSumAtrIX — APPROVED** [review]

**Pull request merged**